Makefile

```diff
@@ -22,5 +22,8 @@ test:
 coverage:
 	python3 -m pytest --cov=smartcards --cov-report term-missing --cov-fail-under=100 tests/
 
+typecheck:
```

Comment:
uhuh, I'm sure this was VsCode nagging ;)

Reply:
Well I had to add it to CI and you do love your makefiles so… 😝